tests/regression/runner.py

```diff
@@ -17,6 +19,49 @@
 MSG_STATUS_CODE_405 = "[405] Method Not Allowed"
 CURL_EMPTY_REPLY = "000"
 
+TIMES_TOTAL = 100
+
+
+def eval_stat(time_cost, func_name):
+    avg = np.average(time_cost)
+    var = np.var(time_cost)
+    print("average time cost of" + str(func_name) + ":" + str(avg) + " sec")
```

Comment:
The Average Elapsed Time is calculated as the average time it took for all request messages in a set of trace results to receive the endpoint. Not "cost of".